packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -353,11 +353,20 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
     }
   }
 
+  int _onChangedCallId = 0;
```

Comment:
Maybe add a comment here explaining what this variable does, what problem it solves etc?

Reply:
sure

Reply:
@victorsanni i have added a comment, can you please review that.

Reply:
I don't think the current comment is sufficient for someone new trying to understand what problem introducing this variable solves. I'd recommend a comment that is a summary of the expected vs. actual results linked to the issue (i.e "assigning an ID to every call of ... is necessary to avoid a situation where ...").

Reply:
Also, comments should end in a period, as per the style guide.

Reply:
Updated the comment @victorsanni